projects/chaos/inc/relay.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <stdbool.h>
+
+#include "event_queue.h"
+
+// Initializes the relay FSMs.
+void relay_init(bool loopback);
+
+// Updates the relays based on an event.
+bool relay_process_event(const Event *e);
```

Comment:
Can you add a function that creates the appropriate open/close events for the given `RelayId`?

Reply:
I'm not sure why this would be necessary? It seems pretty explicit to just have

```c
Event e = { .id = CHAOS_CAN_RELAY_OPEN, .data = RELAY_ID };
```

These event are only ever programmatically generated in the CAN handler or in tests. This is application level logic so it isn't as necessary to have an API for general use.

Reply:
It helps people trying to understand the code/debug it in the future. Just because it's application level doesn't mean it should be implicit. At the very minimum, comments explaining the intended operation are necessary.

Otherwise, I need to actually go into `relay_fsm.c` to understand what `data` should be. If we ever change how `data` is processed, we'll need to update the event everywhere it's used. My issue with data fields are that there's no standard on how to use them or what they should contain, so raising the appropriate raw events requires implementation knowledge.

Reply:
Done.